Header header
Header image_header
BoundingBox[] bounding_boxes
int32 img_w
int32 img_h
